db/migrations/sdp-migrations/2024-08-20.0-alter-receiver-verifications-table-add-verification-channel.sql

```diff
@@ -0,0 +1,12 @@
+-- +migrate Up
+ALTER TABLE receiver_verifications
+    ADD COLUMN verification_channel message_channel;
```

Comment:
I left this as Nullabe and didn't set a default, because when a verification is first created it doesn't have a verification channel. The channel is added during confirmation.